conduit/data/datasets/audio/base.py

```diff
@@ -65,4 +65,4 @@ def load_waveform(self, index: int) -> Tensor:
     @implements(CdtDataset)
     def _sample_x(self, index: int, *, coerce_to_tensor: bool = False) -> Tensor:
         waveform = self.load_waveform(index)
-        return apply_waveform_transform(waveform)
+        return apply_waveform_transform(waveform, transform=None)
```

Comment:
i don't think that this is used yet, but it was missing a required kwarg

Reply:
there is a transform attribute, isn't there?

Reply:
because the preprocessing is no longer done on the fly, I think that this should now be
```suggestion
        return self.load_spectrogram(path)
```

Reply:
or index, whatever

Reply:
There is no such method as `load_spectrogram` currently, btw, right?